tests/block/033

```diff
@@ -7,7 +7,7 @@
 # can be covered
 
 . tests/block/rc
-. common/ublk
+. tests/ublk/rc
```

Comment:
This creates dependency from tests/block to tests/ublk, which is not good. Instead, I suggest to take two actions:
1) move the UBLK_PROG declarations below from tests/ublk/rc to common/ublk
```
if which rublk > /dev/null 2>&1; then
       export UBLK_PROG="src/rublk_wrapper.sh"
else
       export UBLK_PROG="src/miniublk"
fi
```
2) Replace ublk_prog in tests/block/033 with UBLK_PROG.

Reply:
Thanks updated as suggested, will open another PR for the misc fix.

Reply:
[@ming1](https://github.com/ming1)  Please help review this change, thanks.